app/presenters/stream_tag_presenter.rb

```diff
@@ -0,0 +1,29 @@
+#   Copyright (c) 2016, Diaspora Inc.  This file is
+#   licensed under the Affero General Public License version 3 or later.  See
+#   the COPYRIGHT file.
+
+class StreamTagPresenter < BasePresenter
```

Comment:
I think I prefer `TagStreamPresenter`, `StreamTagPresenter` sounds like it's for a special kind of tag, a "stream tag", while it's a special kind of stream, a "tag stream".